core-models/.scalafmt.conf

```
@@ -0,0 +1 @@
+maxColumn = 160
```

shall we define this on a higher level?

---

please read the disclaimer :)

---

Really, 160? I thought Github works nice with 120 🙂

---

```suggestion
maxColumn = 120
```

I use IDEA on a 15" and usually have the project structure opened on the left which leaves me with about 2/3 of my screen for the code view at font size 12. In this setup, I cannot see more than 120 chars.

On 11" or 13" reading diffs at 160 columns is practically impossible. I personally lean towards the legacy ideal of 80, where 120 is an upper limit that should be used for everything that cannot be meaningfully formatted to 80.